sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractConfiguration.java

```diff
@@ -113,24 +117,25 @@ private String getAnalyzerReportDir() {
   }
 
   private static boolean validateOutputDir(Path analyzerOutputDir, boolean silent) {
+    String path = analyzerOutputDir.toString();
     try {
       if (!analyzerOutputDir.toFile().exists()) {
-        ifNotSilent(silent, () -> LOG.warn("Analyzer working directory does not exist: " + analyzerOutputDir.toAbsolutePath() + MSG_SUFFIX));
+        ifNotSilent(silent, () -> LOG.warn("Analyzer working directory does not exist: '{}'" + MSG_SUFFIX, path));
         return false;
       }
 
       try (DirectoryStream<Path> files = Files.newDirectoryStream(analyzerOutputDir, protoFileFilter())) {
         long count = StreamSupport.stream(files.spliterator(), false).count();
         if (count == 0) {
-          ifNotSilent(silent, () -> LOG.warn("Analyzer working directory contains no .pb file(s)" + MSG_SUFFIX));
+          ifNotSilent(silent, () -> LOG.warn("Analyzer working directory '{}' contains no .pb file(s)" + MSG_SUFFIX, path));
           return false;
         }
 
-        ifNotSilent(silent, () -> LOG.debug("Analyzer working directory contains " + count + " .pb file(s)"));
+        ifNotSilent(silent, () -> LOG.debug("Analyzer working directory '{}' contains {} .pb file(s)", path, count));
         return true;
       }
     } catch (IOException e) {
-      throw new IllegalStateException("Could not check for .pb files in " + analyzerOutputDir.toAbsolutePath(), e);
+      throw new IllegalStateException("Could not check for .pb files in '" + path + "'", e);
```

Comment:
Looks like you've changed all of the other messages to not use string concatenation, except this one.

Reply:
good catch